src/h5web/providers/jupyter/devApi.ts

```diff
@@ -1,10 +1,88 @@
 import { JupyterStableApi } from './api';
+import type { Attribute } from '../models';
+import type {
+  JupyterBaseEntity,
+  JupyterContentGroupResponse,
+  JupyterMetaGroupResponse,
+  JupyterMetaResponse,
+} from './models';
+import { assertGroupContent, convertDtype } from './utils';
+
+interface DevJupyterAttrMeta {
+  name: string;
+  dtype: string;
+  shape: number[];
+}
+
+interface DevJupyterMetaResponse extends JupyterMetaResponse {
+  attributes: DevJupyterAttrMeta[];
+}
+
+interface DevJupyterMetaGroupResponse extends JupyterMetaGroupResponse {
+  attributes: DevJupyterAttrMeta[];
+  childrenCount: number;
+}
```

Comment:
I introduced a few additional types. I thought that keeping them in the same file would ease the updating of the stable API.